### DISCUSSION TITLE
Cron Job Timezone Issue

### DISCUSSION THREAD

**Discussion by Tech-MGRV** [supabase/supabase#7892] [Feature Requests] [answered]
I made cron job as below.
And it works on UTC +0
Is there a way to change timezone for cron job?

===

select
  cron.schedule(
      'config',
      '0 0 * * *',
      $$
      update ...
      $$
  )

**Comment by GaryAustin1** [comment, marked as answer]
This seems to indicate that is how it works and no option to change.  Just from googling...
https://github.com/citusdata/pg_cron/issues/16

**Comment by vongohren** [comment]
Things have changed now: https://github.com/citusdata/pg_cron/issues/16#issuecomment-1415947296

Can we unanswer and ask for an upgrade from supabase to support this somehow?

**Reply by GaryAustin1** [reply]
https://github.com/supabase/postgres/pull/634
Supabase tried to upgrade and had issues and backed it out.

**Reply by vongohren** [reply]
Ah, I guess it will either take time or never come then :P ?

**Reply by GaryAustin1** [reply]
I don't know what their plan is.  You could generate an issue in supabase/postgres and link that along with your needs.  That is now two features missing (second resolution the other).

**Reply by vongohren** [reply]
There are an issue for update pg_cron, so just piggybacking on that. See you have been there as well :D

**Reply by nilsreichardt** [reply]
How can I use https://github.com/citusdata/pg_cron/issues/16#issuecomment-1415947296 in Supabase?

**Reply by Ayfri** [reply]
Any update ?
